TurboWarp/Better timer/Better timer.js

```diff
@@ -0,0 +1,290 @@
+
+          "---",
+
+          {
+            opcode: "pauseTimer",
+            blockType: Scratch.BlockType.COMMAND,
+            extensions: ["colours_sensing"],
+            text: "暂停计时器[TIMER]",
+            arguments: {
+              TIMER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "timer",
+              },
+            },
+          },
+          {
+            opcode: "resumeTimer",
+            blockType: Scratch.BlockType.COMMAND,
+            extensions: ["colours_sensing"],
+            text: "恢复计时器[TIMER]",
+            arguments: {
+              TIMER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "timer",
+              },
+            },
+          },
+
+          "---",
+
+          {
+            opcode: "setTimer",
+            blockType: Scratch.BlockType.COMMAND,
+            extensions: ["colours_sensing"],
+            text: "设置计时器[TIMER]为[NUM]",
+            arguments: {
+              TIMER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "timer",
+              },
+              NUM: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: "10",
+              },
+            },
+          },
+          {
+            opcode: "changeTimer",
+            blockType: Scratch.BlockType.COMMAND,
+            extensions: ["colours_sensing"],
+            text: "更改计时器[TIMER]将其增加[NUM]",
+            arguments: {
+              TIMER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "timer",
+              },
+              NUM: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: "10",
+              },
+            },
+          },
+
+          "---",
+
+          {
+            opcode: "removeTimer",
+            blockType: Scratch.BlockType.COMMAND,
+            extensions: ["colours_sensing"],
+            text: "删除计时器[TIMER]",
+            arguments: {
+              TIMER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "timer",
+              },
+            },
+          },
+          {
+            opcode: "removeTimers",
+            blockType: Scratch.BlockType.COMMAND,
+            extensions: ["colours_sensing"],
+            text: "删除全部计时器",
+          },
+          {
+            opcode: "timerExists",
+            blockType: Scratch.BlockType.BOOLEAN,
+            extensions: ["colours_sensing"],
+            text: "计时器[TIMER]存在吗？",
+            arguments: {
+              TIMER: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "timer",
+              },
+            },
+          },
+          {
+            opcode: "listExistingTimers",
+            blockType: Scratch.BlockType.REPORTER,
+            extensions: ["colours_sensing"],
+            text: "列出所有计时器",
+            disableMonitor: false,
+          },
```

Comment:
To add an I10n translation, use the translate method in the global scratch object or in the incoming scratch object, rather than modifying the relevant text directly. This is no different from no Chinese translation, which is unfriendly to overseas users